ios/CustomerioReactnative.swift

```diff
@@ -113,5 +96,15 @@ f     */
         }
         CustomerIO.shared.screen(name: name, data: body)
     }
+    
+    /**
+        Intialize in-app using customerio package
+     */
+    @objc(initializeInApp:)
+    func initializeInApp(organizationId: String) -> Void{
```

Comment:
Can't we force in-app at the time of sdk initialization? Android SDK doesn't really support adding and setting up modules later.

Or if needed, we can merge this and think of more flexible solution later for this. Thoughts? 🤔

Reply:
I see. We can get organisation Id (used for in-app) while initialization, not a deal but I am wondering why this is always a difference in iOS & Android. 🤔 

If this is a difference in iOS & Android implementation, then I would prefer to make a fix immediately rather than making the change with Android release.
Reason - This would be an implementation change and then users will have to make changes in their code too. If we know that this is going to come up later then let's fix it right now.

Reply:
Agreed. The major challenge here is these options in Android are only exposed via builder object, which makes sense to have as they are one time. But here in react, the pattern is allowing to make calls after the initialization.

If we can somehow add builder pattern in the react native initialization, that will overcome this problem and might help reducing a lot more upcoming challenges for Android. 💭

Reply:
Makes sense. I have updated the package to keep the implementation consistent across iOS & Android.

Reply:
Thank you for the quick changes 👍🏻